.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+<!-- TODO: Replace the title below -->
+# Your Title
+
+## Description
+
+<!-- TODO: Write a description of the proposed changes -->
+
+### Type of Change
+
+<!-- TODO: Fill in the brackets with an `x` next to all types that apply to the proposed changes -->
+- [ ] Feature
+- [ ] Bug fix
+- [ ] Documentation
+
+### Test Coverage
+
+<!-- TODO: Fill in the test coverage -->
+`0%`
```

Comment:
You'll probably want to rebase off of the most recent version of PR #25 so that there aren't any conflicts.